ora_migrator--1.1.0.sql

```diff
@@ -98,6 +122,42 @@ $$DECLARE
          '  AND col.owner NOT IN (' || sys_schemas || E')'
       ')'', max_long ''%s'', readonly ''true'')';
 
+   column_comments text := E'CREATE FOREIGN TABLE %I.column_comments (\n'
+      '   schema        text    NOT NULL,\n'
+      '   table_name    text    NOT NULL,\n'
+      '   column_name   text    NOT NULL,\n'
+      '   comment       text    NOT NULL\n'
+      ') SERVER %I OPTIONS (table ''('
+         'SELECT col.owner,\n'
+         '       col.table_name,\n'
+         '       col.column_name,\n'
+         '        cc.comments\n'
+         'FROM dba_tab_columns col\n'
+         'JOIN dba_col_comments cc\n'
+         '  ON cc.owner = col.owner\n'
+         ' AND cc.table_name = col.table_name\n'
+         ' AND cc.column_id = col.column_id\n'
+         '   JOIN (SELECT owner, table_name\n'
+         '            FROM dba_tables\n'
+         '            WHERE owner NOT IN (' || sys_schemas || E')\n'
+         '              AND temporary = ''''N''''\n'
+         '              AND secondary = ''''N''''\n'
+         '              AND nested    = ''''NO''''\n'
+         '              AND dropped   = ''''NO''''\n'
+         '         UNION SELECT owner, view_name\n'
+         '            FROM dba_views\n'
+         '            WHERE owner NOT IN (' || sys_schemas || E')\n'
+         '        ) tab\n'
+         '      ON tab.owner = col.owner AND tab.table_name = col.table_name\n'
+         'WHERE (col.owner, col.table_name)\n'
+         '     NOT IN (SELECT owner, mview_name\n'
+         '             FROM dba_mviews)\n'
+         '  AND (col.owner, col.table_name)\n'
+         '     NOT IN (SELECT log_owner, log_table\n'
+         '             FROM dba_mview_logs)\n'
+         '  AND col.owner NOT IN (' || sys_schemas || E')'
```

Comment:
I don't think this line is necessary. Such owners are already removed in the `tab` subquery, and `tab.owner = col.owner`.

I realize that you copied that from my code, so it is my fault.

Reply:
Removed, thanks.